installer/common/install-cloudflow.sh

```diff
@@ -38,7 +39,19 @@ fi
 
 # Adjust the user rights
 echo "Creating admin cluster role binding for modifying Spark role"
-kubectl create clusterrolebinding cluster-admin-binding --clusterrole cluster-admin --user "$(gcloud config get-value account)" > /dev/null
+
+case $CLUSTER_TYPE in
+ gke)
+    kubectl create clusterrolebinding cluster-admin-binding --clusterrole cluster-admin --user "$(gcloud config get-value account)" >/dev/null
+    ;;
+ eks)
+    kubectl create clusterrolebinding cluster-admin-binding --clusterrole cluster-admin --user "$(aws iam get-user | jq -r .User.UserName)" >/dev/null
+    ;;
+ *)
+    print_error_message "Unknown cluster type: $CLUSTER_TYPE"
+    exit 1
+    ;;
+esac
 
 echo "Installing NFS Server"
```

Comment:
We really only need to install NFS server on GKE because GKE doesn't come with a native storage class provisioner that supports RWM access. On EKS and all other distributions we have ever tried there is always at least one RWM storage class provisioner available. So we should probably make NFS a special case for GKE.

Reply:
Hmmm, iinteresting. I think we actually need NFS for EKS too. Please see the following table:

https://kubernetes.io/docs/concepts/storage/persistent-volumes/

EBS doesn't support  RWM, however maybe we can add support for AWS EFS?

https://github.com/kubernetes-incubator/external-storage/tree/master/aws/efs

This change mught be a bigger one thoug. Hwever this is preventing from testing the FileIngres :(

Reply:
I agree that AWS RWM can/should be a separate PR, especially if this is blocking you.